.cloudbees/workflows/workflow.yaml

```diff
@@ -33,16 +38,31 @@ jobs:
     - name: docker build and publish
       uses: cloudbees-io/kaniko
       with:
-        destination: registry.saas-dev.beescloud.com/staging/helm-install-action:${{ cloudbees.scm.sha }}${{ cloudbees.scm.branch == 'main' && ',registry.saas-dev.beescloud.com/staging/helm-install-action:1.1.4,registry.saas-dev.beescloud.com/staging/helm-install-action:latest' || '' }}
+        destination: ${{ vars.staging_action_registry }}/actions/helm-install-action:${{ cloudbees.scm.sha }}${{ cloudbees.scm.branch == 'main' && format(',{0}/actions/helm-install-action:1.1.5,{0}/actions/configure-git-global-credentials:latest', vars.staging_action_registry)  || format(',{0}/actions/helm-install-action:{1}', vars.staging_action_registry, cloudbees.version) }}
+        context: ${{ cloudbees.workspace }}
         labels: maintainer=sdp-pod-3,email=engineering@cloudbees.com
   install-test:
     needs: build
     steps:
-    - name: write kube config
-      uses: docker://alpine:3.17
-      run: |
-        mkdir -p ${HOME}/.kube
-        echo -e "${{ secrets.STAGING_CONFIG }}" > ${HOME}/.kube/config
+     - name: Login to AWS
+        uses: https://github.com/cloudbees-io/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: ${{ vars.oidc_staging_iam_role }}
+          role-duration-seconds: "3600"
+    - id: ecrconfig
+      name: Configure container registry for staging ECR
+      uses: cloudbees-io/configure-ecr-credentials@v0
+    - id: dockerhub
```

Comment:
For the `install-test` you do not need the `dockerhub` login step

Reply:
After discussing with @relaprolu - he used this docker hub login step to use an authenticated DockerHub user for pulling the OCI charts from `registry-1.docker.io` in the steps of this job. In that case @relaprolu, maybe we should also update the `registry` field for the DockerHub login step to use `registry-1.docker.io`?